Specs/addDefaultMatchers.js

```diff
@@ -584,20 +585,20 @@ define([
         var rgba = context.readPixels();
         if (!webglStub) {
             if (expectEqual) {
-                if ((rgba[0] !== expected[0]) ||
-                    (rgba[1] !== expected[1]) ||
-                    (rgba[2] !== expected[2]) ||
-                    (rgba[3] !== expected[3])) {
+                if (!CesiumMath.equalsEpsilon(rgba[0], expected[0], epsilon) ||
+                    !CesiumMath.equalsEpsilon(rgba[1], expected[1], epsilon) ||
+                    !CesiumMath.equalsEpsilon(rgba[2], expected[2], epsilon) ||
+                    !CesiumMath.equalsEpsilon(rgba[3], expected[3], epsilon)) {
```

Comment:
`toReadPixels` uses all four arguments of `equalsEpslion`. Worth doing the same here.

`if (!CesiumMath.equalsEpsilon(rgba[0], expected[0], 0, epsilon) ||`